packages/taco/src/conditions/context/context.ts

```diff
@@ -41,18 +40,16 @@ const ERR_AUTH_PROVIDER_NOT_NEEDED_FOR_CONTEXT_PARAM = (param: string) =>
 
 type AuthProviderType =
   | typeof EIP4361AuthProvider
-  | typeof SingleSignOnEIP4361AuthProvider
-  | typeof EIP1271AuthProvider;
+  | typeof EIP1271AuthProvider
+  | typeof SingleSignOnEIP4361AuthProvider;
 
-const EXPECTED_AUTH_PROVIDER_TYPES: Record<string, AuthProviderType> = {
-  [USER_ADDRESS_PARAM_DEFAULT]: EIP4361AuthProvider,
-  [USER_ADDRESS_PARAM_EXTERNAL_EIP4361]: SingleSignOnEIP4361AuthProvider,
-  [USER_ADDRESS_PARAM_EIP1271]: EIP1271AuthProvider,
+const EXPECTED_AUTH_PROVIDER_TYPES: Record<string, AuthProviderType[]> = {
+  [USER_ADDRESS_PARAM_DEFAULT]: [EIP4361AuthProvider, EIP1271AuthProvider],
```

Comment:
I think we want to allow single-sign on for EIP4361 as well.

```suggestion
  [USER_ADDRESS_PARAM_DEFAULT]: [EIP4361AuthProvider, EIP1271AuthProvider, SingleSignOnEIP4361AuthProvider],
```